st/m0d-signal-test.sh

```diff
@@ -49,14 +49,14 @@ test_for_signal()
     local sig=$1
     echo "------------------ Configuring Motr for $sig test ------------------"
     systemctl start motr-mkfs
+    local cursor=$(journalctl --show-cursor -n0 | grep -e '-- cursor:' | sed -e 's/^-- cursor: //')
```

Comment:
AFAIK, motr-server@ios1 service will start when trigger "systemctl start motr".

Reply:
That's what I want: 
>we are capturing cursor before ""systemctl start motr".

This is the cursor.
And later, using 'journalctl' from that cursor to get the latest log: from that point of time to current.
```
 60     while ! journalctl -c "$cursor" -l -u motr-server@ios1 | grep -q 'Press CTRL+C to quit'; do
 61         sleep 1
 62     done
```

Reply:
It is working with latest CI result 149 & 150.

Reply:
Thank you for the confirmation.